src/explore-education-statistics-frontend/src/modules/find-statistics/components/__tests__/SortControls.test.tsx

```diff
@@ -0,0 +1,59 @@
+import SortControls from '@frontend/modules/find-statistics/components/SortControls';
+import { render, screen, within } from '@testing-library/react';
+import React from 'react';
+import noop from 'lodash/noop';
+
+let mockIsMedia = false;
+jest.mock('@common/hooks/useMedia', () => ({
+  useMobileMedia: () => {
+    return {
+      isMedia: mockIsMedia,
+    };
+  },
+}));
+
+describe('SortControls', () => {
+  test('renders the desktop version', () => {
+    render(
+      <SortControls initialValues={{ sortBy: 'newest' }} onChange={noop} />,
+    );
+
+    const sortGroup = within(
+      screen.getByRole('group', { name: 'Sort results' }),
+    );
+    const sortOptions = sortGroup.getAllByRole('radio');
+    expect(sortOptions).toHaveLength(3);
+    expect(sortOptions[0]).toEqual(sortGroup.getByLabelText('Newest'));
+    expect(sortOptions[0]).toBeChecked();
+    expect(sortOptions[1]).toEqual(sortGroup.getByLabelText('Oldest'));
+    expect(sortOptions[1]).not.toBeChecked();
+    expect(sortOptions[2]).toEqual(sortGroup.getByLabelText('A to Z'));
+    expect(sortOptions[2]).not.toBeChecked();
+
+    expect(screen.queryByLabelText('Sort results')).not.toBeInTheDocument();
+  });
+
```

Comment:
We should have a test to check that providing a different initial `sortBy` sets a different selected value (for desktop and mobile).